validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
@lilleyse Just a quick question for the first two test cases: what would I put within the function once I've received the parsed JSON such as in `readTileSpec`?